media-radio/cqrlog/cqrlog-2.4.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="CQRLOG is an advanced ham radio logger based on MySQL database"
+HOMEPAGE="https://www.cqrlog.com/ https://github.com/ok2cqr/cqrlog"
+SRC_URI="https://github.com/ok2cqr/cqrlog/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-2"
+
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND=">=dev-lang/lazarus-1.8.0
+	>=dev-lang/fpc-3.0.2"
+
+RDEPEND="${DEPEND}
+	virtual/mysql[server]
+	dev-libs/atk
+	dev-libs/glib
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf
+	x11-libs/gtk+
+	x11-libs/libX11
+	x11-libs/pango"
+
+PATCHES=(
+	"${FILESDIR}/${PV}-makefile.patch"
```

Comment:
Please also include the package name in the patch file, cqrlog-2.4.0-fix-installdirs.patch

Reply:
I still can't reproduce this error, even on a pretty old and weak machine (1 CPU, 2GB RAM). The failure of creating a new thread looks like some resource insufficiency issue. I've made some changes in Makefile (through patch), can you try to compile it again, please?

About the package file name - I used the same pattern as previous versions did. I'm going to rename it after solving the current problem with compile, OK?

Reply:
Sure. Can you show me your `emerge --info` and **full** emerge.log for this package? I've tried different things, like gcc-9, `MAKEOPTS="-j1"` to no avail.

Reply:
Okay. I managed to build this on my **stable** system, but this fails on my **unstable** one even **with** gcc-9. That's problematic, as it has `~` KEYWORDS it **needs** to work on unstable system. Something in the newer toolchain breaks it, most likely glibc somehow, or *fpc* which has been built with newer gcc+glibc. 

This will be a harder one to debug, but it needs to work on unstable systems. You really need to head into upstream issues and try to find solutions to latest build issues. And do check how other up-to-date distributions deal with this particular package.

Reply:
So I must set up my unstable system now to see what's going on. Another new thing for me :-)

Reply:
You can do that inside a chroot, or set up a container (docker, lxd etc)

Reply:
Thank you for the tips. I'm going to use kvm/qemu for making possible fiddling with resources.

Reply:
So I'm now confused a bit. Does the "unstable system" mean the system completely built with "ACCEPT_KEYWORDS="~amd64"? I've done so and cqrlog builds there like a charm.

Reply:
Yes it does. Did you update your system? So you have 
```
sys-libs/glibc-2.32-r7:2.2::gentoo
sys-devel/gcc-10.2.0-r5:10::gentoo
```
installed? 

If so, looks like I may have to get my hands dirty on this one.

@thesamesam could you test this?

Reply:
I have

```
sys-libs/glibc-2.32-r6::gentoo
sys-devel/gcc-10.2.0-r5::gentoo

```
installed, but also 

```
sys-devel/gcc-9.3.0-r2::gentoo
```

which was used to build all the packages.

So I've switched to gcc-10 and the cqrlog compiles as well.

Reply:
Yesterday I updated the system, so now I have

```
sys-libs/glibc-2.32-r7::gentoo
sys-devel/gcc-10.2.0-r5::gentoo

```
and cqrlog still compiles.

Reply:
Hello, is there any way now, how can I help making this version bump mergable?